.heroku/check_change_traffic_control.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import subprocess
+import sys
+from calendar import timegm
+from datetime import datetime
+
+import jwt
+import pydantic
+import requests
+
+logger = logging.getLogger()
+
+
+class CTCSettings(pydantic.BaseSettings):
+    ctc_url: str
+    ctc_auth_key: str
```

Comment:
Change this to use metaci_release_webhook_url and metaci_release_webhook_auth_key so we don't have to duplicate

Reply:
@davisagli not sure you I correctly interpreted what you wanted, let me know.